daffodil-core/src/main/java/org/apache/daffodil/api/debugger/InteractiveDebuggerRunner.java

```diff
@@ -20,30 +20,38 @@
 import org.apache.daffodil.runtime1.debugger.InteractiveDebugger;
 
 /**
- * Interactive Debugger Runner interface
+ * Interactive Debugger Runner interface to run interactive debuggers. Must be implemented
```

Comment:
> Interactuve Debugger Runner interface to run the built-in Daffodil interactive debugger.

I think "Must be implemented for custom debugger runners" is a bit confusing, and almost implies custom debuggers must use this, but they don't. Custom debuggers (like VS COde debugger) can implement the "Debugger" interface and never have to use the InteractiveDebugger interface. This interface is only for people that want to use the built-in debugger but want to control it with custom input/output methods.